data/856/719/73/85671973.geojson

```diff
@@ -577,14 +587,14 @@
         "ind",
         "eng"
     ],
-    "wof:lastmodified":1617130624,
+    "wof:lastmodified":1623109425,
     "wof:name":"Aceh",
     "wof:parent_id":85632203,
     "wof:placetype":"region",
     "wof:placetype_alt":[],
     "wof:placetype_local":"autonomous province",
```

Comment:
Should this value also change to be consistent with the preferred placetype change above?

Reply:
Yeah, good catch.. I'll double check the `wof:placetype_local` properties in this PR.